docs/intro.md

```diff
@@ -56,7 +56,7 @@ You can build this book locally on the command line via the following steps:
 2. Install the Python libraries needed to run the code in this particular example:
 
     ```
-    conda install numpy matplotlib
+    conda install numpy scipy matplotlib sympy pandas
```

Comment:
I think instead of manually specifying the dependencies, we should tell people to create
a conda environment from the `environment.yml` file in the repository, so something like:

```suggestion
    cd quantecon-mini-example
    conda create env -f environment.yml
```